.asf.yaml

```diff
@@ -50,8 +50,38 @@ github:
     main:
       required_pull_request_reviews:
         required_approving_review_count: 1
+      required_status_checks:
+        strict: true  # Require branches to be up to date before merging
```

Comment:
Can we not require up to date branches before we have enabled the merge queue? Otherwise that will slow down merging of PRs as we'll have a bottleneck / race for each PR that wants to merge

Once there is an automated queue it makes sense because the queue handles the race